src/api/job.js

```diff
@@ -51,4 +51,33 @@ router.post('/', async (request, response) => {
   }
 });
 
+router.put('/', async (request, response) => {
+  try {
+    const {id} = request.body;
+    // eslint-disable-next-line unicorn/no-fn-reference-in-iterator
+    const isJobExist = await Job.find(id);
```

Comment:
What's the result of `Job.find(id)` function? Does it an object? If so, the name `isJobExist` doesn't make sense because it's a boolean name.